app/controllers/idv/document_capture_controller.rb

```diff
@@ -72,6 +73,7 @@ def self.step_info
 
     def confirm_hybrid_handoff_complete
       return if idv_session.flow_path.present?
+      return if idv_session.skip_doc_auth == true
```

Comment:
Please just set flow_path to 'standard' rather than complicating the logic here. Let me know if there was something you were trying to fix by adding this.

Reply:
I deleted this line. Resolved with commit dc71edf32fe67393b17a6091487ed0328ea03fa1